### ISSUE TITLE
Templates broken with AppDaemon 4.4.1 / 0.12.0

### ISSUE THREAD

**Issue by ntls2000** [joBr99/nspanel-lovelace-ui#847]
### PROBLEM DESCRIPTION
Hi, i have install appdaemon on docker and i have a error on date and time.
He add template to None and create a error
All information from sensor are OK

no timeAdditionalTemplate in config.yaml

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Provide your apps.yaml (or your config):
```
---
nspanel-1:
  module: nspanel-lovelace-ui
  class: NsPanelLovelaceUIManager
  config:
    panelRecvTopic: "tele/nspanel01_E7F10C/RESULT"
    panelSendTopic: "cmnd/nspanel01_E7F10C/CustomSend" 
    sleepTimeout: 10
    sleepBrightness:
      - time: "7:00:00"
        value: 10
      - time: "22:30:00"
        value: 1
    locale: "fr_FR"
    screensaver:
      type: screensaver2
      entities:
        - entity: weather.lunel
        - entity: sensor.tasmota_analog_temperature1
          icon: mdi:home-thermometer-outline        
        - entity: sensor.edf_pinst
          icon: mdi:home-lightning-bolt-outline
        - entity: sensor.edf_price_tax_month
          icon: mdi:currency-eur 
        - entity: sensor.lunel_cloud_cover
          name: Nuage
          icon: mdi:weather-partly-cloudy
          type: 1                                      
        - entity: sensor.lunel_rain_chance
          icon: mdi:weather-rainy
          name: Pluie
          type: 2
        - entity: sensor.lunel_wind_gust
          icon: mdi:weather-windy-variant
          name: Rafale
          type: 3
        - entity: sensor.lunel_wind_speed
          icon: mdi:weather-windy-variant
          name: Vent
        - entity: sensor.lunel_uv
          name: UV
          icon: mdi:sunglasses
        - entity: sensor.34_weather_alert
          icon: mdi:flag
          name: Niveau
          color:
              "Jaune": [255,255,0]
              "Vert": [0,255,0]
              "Rouge": [255,0,0]
      doubleTapToUnlock: True
      statusIcon1:
        entity: switch.tasmota_c4dd57e7f10c_switch_relay_0
        icon: mdi:light-switch
      statusIcon2:
        entity: switch.tasmota_c4dd57e7f10c_switch_relay_1
        icon: mdi:light-switch
    cards:
      - type: cardEntities
        title: Test Entities Card
        entities:
          - entity: light.lavabo


```
- [ ] Go to Settings>Add-ons>AppDaemon>Log (or the output of your TS script) and then, provide the output of the log during your issue/bug occurs:
```
  Log output here:

2023-04-09 10:35:50.062411 INFO nspanel-1: Sending MQTT Message: weatherUpdate~~~���~38066~Lunel~13.0��C~~~���~17299~Tasmota ANALOG Temperature1~27.4  ~~~���~17299~edf_pinst~509.09 W~~~���~17299~edf_price_tax_month~12.5035916 EUR/kWh~~~���~17299~Nuage~30 %~~~���~17299~Pluie~0 %~~~���~17299~Rafale~0 km/h~~~���~17299~Vent~7 km/h~~~���~17299~UV~6 ~~~���~2016~Niveau~Vert 
2023-04-09 10:35:50.072344 INFO nspanel-1: Sending MQTT Message: statusUpdate~���~65222~���~65222~~
2023-04-09 10:35:50.095290 INFO nspanel-1: Sending MQTT Message: timeout~10
2023-04-09 10:35:50.137646 INFO nspanel-1: Sending MQTT Message: weatherUpdate~~~���~38066~Lunel~13.0��C~~~���~17299~Tasmota ANALOG Temperature1~27.4  ~~~���~17299~edf_pinst~509.09 W~~~���~17299~edf_price_tax_month~12.5036036 EUR/kWh~~~���~17299~Nuage~30 %~~~���~17299~Pluie~0 %~~~���~17299~Rafale~0 km/h~~~���~17299~Vent~7 km/h~~~���~17299~UV~6 ~~~���~2016~Niveau~Vert 
2023-04-09 10:35:50.145546 INFO nspanel-1: Sending MQTT Message: statusUpdate~���~65222~���~65222~~
2023-04-09 10:36:00.966056 ERROR HASS: ------------------------------------------------------------
2023-04-09 10:36:00.966461 ERROR HASS: Unexpected error during call_plugin_service()
2023-04-09 10:36:00.966856 ERROR HASS: Service: default.template.render Arguments: {'template': None}
2023-04-09 10:36:00.967250 ERROR HASS: ------------------------------------------------------------
2023-04-09 10:36:00.967919 ERROR HASS: Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/appdaemon/plugins/hass/hassplugin.py", line 556, in call_plugin_service
    r = await self.session.post(api_url, json=data)
  File "/usr/local/lib/python3.10/site-packages/aiohttp/client.py", line 508, in _request
    req = self._request_class(
  File "/usr/local/lib/python3.10/site-packages/aiohttp/client_reqrep.py", line 305, in __init__
    self.update_host(url)
  File "/usr/local/lib/python3.10/site-packages/aiohttp/client_reqrep.py", line 364, in update_host
    raise InvalidURL(url)
aiohttp.client_exceptions.InvalidURL: /api/template

2023-04-09 10:36:00.968304 ERROR HASS: ------------------------------------------------------------
2023-04-09 10:36:00.971004 INFO nspanel-1: Sending MQTT Message: time~10:36~None


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots/pictures to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._
_Please note here in case you are using ioBroker_

### PANEL / FIRMWARE VERION
Last version EU

**Comment by joBr99** [comment]
Can you restart the appdaemon addon and post the startup logs?

**Comment by ntls2000** [comment]
```
2023-04-09 11:22:40.038630 INFO AppDaemon: Loading Plugin MQTT using class MqttPlugin from module mqttplugin
2023-04-09 11:22:40.064997 INFO MQTT: MQTT Plugin Initializing
2023-04-09 11:22:40.065270 INFO MQTT: Using 'appdaemon/status' as Will Topic
2023-04-09 11:22:40.065419 INFO MQTT: Using 'appdaemon/status' as Birth Topic
2023-04-09 11:22:40.066115 INFO AppDaemon: Initializing HTTP
2023-04-09 11:22:40.066575 INFO AppDaemon: Using 'ws' for event stream
2023-04-09 11:22:40.099326 INFO AppDaemon: Starting API
2023-04-09 11:22:40.103656 INFO AppDaemon: Starting Admin Interface
2023-04-09 11:22:40.104186 INFO AppDaemon: Starting Dashboards
2023-04-09 11:22:40.118318 INFO HASS: Connected to Home Assistant 2023.4.1
2023-04-09 11:22:40.125255 INFO MQTT: Connected to Broker at URL 192.168.0.3:1883
2023-04-09 11:22:40.178423 INFO AppDaemon: Got initial state from namespace mqtt
2023-04-09 11:22:40.178899 INFO MQTT: MQTT Plugin initialization complete
2023-04-09 11:22:40.201710 INFO HASS: Evaluating startup conditions
2023-04-09 11:22:40.205582 INFO AppDaemon: App 'nspanel-1' added
2023-04-09 11:22:40.206886 INFO AppDaemon: Found 1 active apps
2023-04-09 11:22:40.207252 INFO AppDaemon: Found 0 inactive apps
2023-04-09 11:22:40.207554 INFO AppDaemon: Found 0 global libraries
2023-04-09 11:22:40.207853 INFO AppDaemon: Starting Apps with 1 workers and 1 pins
2023-04-09 11:22:40.208864 INFO AppDaemon: Running on port 5050
2023-04-09 11:22:40.213786 INFO HASS: Startup condition met: hass state=RUNNING
2023-04-09 11:22:40.214156 INFO HASS: All startup conditions met
2023-04-09 11:22:40.249959 INFO AppDaemon: Got initial state from namespace default
2023-04-09 11:22:41.436153 INFO AppDaemon: New client Admin Client connected
2023-04-09 11:22:42.215582 INFO AppDaemon: Scheduler running in realtime
2023-04-09 11:22:42.223769 INFO AppDaemon: Adding /conf/apps to module import path
2023-04-09 11:22:42.224404 INFO AppDaemon: Adding /conf/apps/nspanel-lovelace-ui to module import path
2023-04-09 11:22:42.224944 INFO AppDaemon: Adding /conf/apps/nspanel-lovelace-ui/luibackend to module import path
2023-04-09 11:22:42.225464 INFO AppDaemon: Adding /conf/apps/nspanel-lovelace-ui/luibackend/translations to module import path
2023-04-09 11:22:42.226065 INFO AppDaemon: Adding /conf/apps/nspanel-lovelace-ui/luibackend/translations/backend to module import path
2023-04-09 11:22:42.226814 INFO AppDaemon: Adding /conf/apps/nspanel-lovelace-ui/luibackend/translations/frontend to module import path
2023-04-09 11:22:42.241003 INFO AppDaemon: Loading App Module: /conf/apps/nspanel-lovelace-ui/nspanel-lovelace-ui.py
2023-04-09 11:22:42.349860 WARNING AppDaemon: No app description found for: /conf/apps/nspanel-lovelace-ui/luibackend/apis.py - ignoring
2023-04-09 11:22:42.351911 WARNING AppDaemon: No app description found for: /conf/apps/nspanel-lovelace-ui/luibackend/config.py - ignoring
2023-04-09 11:22:42.353611 WARNING AppDaemon: No app description found for: /conf/apps/nspanel-lovelace-ui/luibackend/controller.py - ignoring
2023-04-09 11:22:42.355297 WARNING AppDaemon: No app description found for: /conf/apps/nspanel-lovelace-ui/luibackend/helper.py - ignoring
2023-04-09 11:22:42.357011 WARNING AppDaemon: No app description found for: /conf/apps/nspanel-lovelace-ui/luibackend/icons.py - ignoring
2023-04-09 11:22:42.358846 WARNING AppDaemon: No app description found for: /conf/apps/nspanel-lovelace-ui/luibackend/icon_mapping.py - ignoring
2023-04-09 11:22:42.360599 WARNING AppDaemon: No app description found for: /conf/apps/nspanel-lovelace-ui/luibackend/localization.py - ignoring
2023-04-09 11:22:42.362315 WARNING AppDaemon: No app description found for: /conf/apps/nspanel-lovelace-ui/luibackend/mqtt.py - ignoring
2023-04-09 11:22:42.364022 WARNING AppDaemon: No app description found for: /conf/apps/nspanel-lovelace-ui/luibackend/pages.py - ignoring
2023-04-09 11:22:42.365714 WARNING AppDaemon: No app description found for: /conf/apps/nspanel-lovelace-ui/luibackend/theme.py - ignoring
2023-04-09 11:22:42.367463 WARNING AppDaemon: No app description found for: /conf/apps/nspanel-lovelace-ui/luibackend/updater.py - ignoring
2023-04-09 11:22:42.368849 INFO AppDaemon: Loading app nspanel-1 using class NsPanelLovelaceUIManager from module nspanel-lovelace-ui
2023-04-09 11:22:42.372888 INFO AppDaemon: Calling initialize() for nspanel-1
2023-04-09 11:22:42.470966 INFO nspanel-1: Starting
2023-04-09 11:22:42.476549 INFO nspanel-1: Input config: {'panelRecvTopic': 'tele/nspanel01_E7F10C/RESULT', 'panelSendTopic': 'cmnd/nspanel01_E7F10C/CustomSend', 'sleepTimeout': 10, 'sleepBrightness': [{'time': '7:00:00', 'value': 10}, {'time': '22:30:00', 'value': 1}], 'locale': 'fr_FR', 'dateAdditionalTemplate': "- {{ states('sun.sun') }}", 'screensaver': {'type': 'screensaver2', 'entities': [{'entity': 'weather.lunel'}, {'entity': 'sensor.tasmota_analog_temperature1', 'icon': 'mdi:home-thermometer-outline'}, {'entity': 'sensor.edf_pinst', 'icon': 'mdi:home-lightning-bolt-outline'}, {'entity': 'sensor.edf_price_tax_month', 'icon': 'mdi:currency-eur'}, {'entity': 'sensor.lunel_cloud_cover', 'name': 'Nuage', 'icon': 'mdi:weather-partly-cloudy', 'type': 1}, {'entity': 'sensor.lunel_rain_chance', 'icon': 'mdi:weather-rainy', 'name': 'Pluie', 'type': 2}, {'entity': 'sensor.lunel_wind_gust', 'icon': 'mdi:weather-windy-variant', 'name': 'Rafale', 'type': 3}, {'entity': 'sensor.lunel_wind_speed', 'icon': 'mdi:weather-windy-variant', 'name': 'Vent'}, {'entity': 'sensor.lunel_uv', 'name': 'UV', 'icon': 'mdi:sunglasses'}, {'entity': 'sensor.34_weather_alert', 'icon': 'mdi:flag', 'name': 'Niveau', 'color': {'Jaune': [255, 255, 0], 'Vert': [0, 255, 0], 'Rouge': [255, 0, 0]}}], 'doubleTapToUnlock': True, 'statusIcon1': {'entity': 'switch.tasmota_c4dd57e7f10c_switch_relay_0', 'icon': 'mdi:light-switch'}, 'statusIcon2': {'entity': 'switch.tasmota_c4dd57e7f10c_switch_relay_1', 'icon': 'mdi:light-switch'}}, 'cards': [{'type': 'cardEntities', 'title': 'Test Entities Card', 'entities': [{'entity': 'light.lavabo'}]}]}
2023-04-09 11:22:42.478251 INFO nspanel-1: Loaded config: {'panelRecvTopic': 'tele/nspanel01_E7F10C/RESULT', 'panelSendTopic': 'cmnd/nspanel01_E7F10C/CustomSend', 'updateMode': 'auto-notify', 'model': 'eu', 'sleepTimeout': 10, 'sleepBrightness': [{'time': '7:00:00', 'value': 10}, {'time': '22:30:00', 'value': 1}], 'screenBrightness': 100, 'defaultBackgroundColor': 'ha-dark', 'sleepTracking': None, 'sleepTrackingZones': ['not_home', 'off'], 'sleepOverride': None, 'locale': 'fr_FR', 'timeFormat': '%H:%M', 'dateFormatBabel': 'full', 'dateFormat': '%A, %d. %B %Y', 'cards': [{'type': 'cardEntities', 'title': 'Test Entities Card', 'entities': [{'entity': 'light.lavabo'}]}], 'screensaver': {'type': 'screensaver2', 'entity': 'weather.example', 'weatherUnit': 'celsius', 'forecastSkip': 0, 'weatherOverrideForecast1': None, 'weatherOverrideForecast2': None, 'weatherOverrideForecast3': None, 'weatherOverrideForecast4': None, 'doubleTapToUnlock': True, 'alternativeLayout': False, 'defaultCard': None, 'key': 'screensaver', 'entities': [{'entity': 'weather.lunel'}, {'entity': 'sensor.tasmota_analog_temperature1', 'icon': 'mdi:home-thermometer-outline'}, {'entity': 'sensor.edf_pinst', 'icon': 'mdi:home-lightning-bolt-outline'}, {'entity': 'sensor.edf_price_tax_month', 'icon': 'mdi:currency-eur'}, {'entity': 'sensor.lunel_cloud_cover', 'name': 'Nuage', 'icon': 'mdi:weather-partly-cloudy', 'type': 1}, {'entity': 'sensor.lunel_rain_chance', 'icon': 'mdi:weather-rainy', 'name': 'Pluie', 'type': 2}, {'entity': 'sensor.lunel_wind_gust', 'icon': 'mdi:weather-windy-variant', 'name': 'Rafale', 'type': 3}, {'entity': 'sensor.lunel_wind_speed', 'icon': 'mdi:weather-windy-variant', 'name': 'Vent'}, {'entity': 'sensor.lunel_uv', 'name': 'UV', 'icon': 'mdi:sunglasses'}, {'entity': 'sensor.34_weather_alert', 'icon': 'mdi:flag', 'name': 'Niveau', 'color': {'Jaune': [255, 255, 0], 'Vert': [0, 255, 0], 'Rouge': [255, 0, 0]}}], 'statusIcon1': {'entity': 'switch.tasmota_c4dd57e7f10c_switch_relay_0', 'icon': 'mdi:light-switch'}, 'statusIcon2': {'entity': 'switch.tasmota_c4dd57e7f10c_switch_relay_1', 'icon': 'mdi:light-switch'}}, 'hiddenCards': [], 'dateAdditionalTemplate': "- {{ states('sun.sun') }}"}
2023-04-09 11:22:42.486302 INFO nspanel-1: Sending MQTT Message: pageType~pageStartup
2023-04-09 11:22:42.504159 INFO nspanel-1: Registering callbacks for the following items: ['light.lavabo', 'weather.example', 'weather.lunel', 'sensor.tasmota_analog_temperature1', 'sensor.edf_pinst', 'sensor.edf_price_tax_month', 'sensor.lunel_cloud_cover', 'sensor.lunel_rain_chance', 'sensor.lunel_wind_gust', 'sensor.lunel_wind_speed', 'sensor.lunel_uv', 'sensor.34_weather_alert', 'switch.tasmota_c4dd57e7f10c_switch_relay_0', 'switch.tasmota_c4dd57e7f10c_switch_relay_1']
2023-04-09 11:22:42.539194 INFO nspanel-1: Started (v4.0.3)
2023-04-09 11:22:42.541377 INFO AppDaemon: App initialization complete
2023-04-09 11:22:42.871228 INFO nspanel-1: MQTT callback for: {'topic': 'tele/nspanel01_E7F10C/RESULT', 'wildcard': None, 'payload': '{"nlui_driver_version":"8"}'}
2023-04-09 11:22:42.872812 INFO nspanel-1: Update Pre-Check failed Tasmota Driver Version: 8 Panel Version: None
2023-04-09 11:22:42.888821 INFO nspanel-1: MQTT callback for: {'topic': 'tele/nspanel01_E7F10C/RESULT', 'wildcard': None, 'payload': '{"CustomRecv":"event,startup,50,eu"}'}
2023-04-09 11:22:42.889989 INFO nspanel-1: Received Message from Screen: event,startup,50,eu
2023-04-09 11:22:42.891179 INFO nspanel-1: Sending MQTT Message: X
2023-04-09 11:22:42.897332 INFO nspanel-1: Update Pre-Check failed Tasmota Driver Version: None Panel Version: 50
2023-04-09 11:22:42.898554 INFO nspanel-1: Startup Event
2023-04-09 11:22:42.900487 ERROR HASS: ------------------------------------------------------------
2023-04-09 11:22:42.900913 ERROR HASS: Unexpected error during call_plugin_service()
2023-04-09 11:22:42.901220 ERROR HASS: Service: default.template.render Arguments: {'template': None}
2023-04-09 11:22:42.901459 ERROR HASS: ------------------------------------------------------------
2023-04-09 11:22:42.903794 ERROR HASS: Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/appdaemon/plugins/hass/hassplugin.py", line 556, in call_plugin_service
    r = await self.session.post(api_url, json=data)
  File "/usr/local/lib/python3.10/site-packages/aiohttp/client.py", line 508, in _request
    req = self._request_class(
  File "/usr/local/lib/python3.10/site-packages/aiohttp/client_reqrep.py", line 305, in __init__
    self.update_host(url)
  File "/usr/local/lib/python3.10/site-packages/aiohttp/client_reqrep.py", line 364, in update_host
    raise InvalidURL(url)
aiohttp.client_exceptions.InvalidURL: /api/template

2023-04-09 11:22:42.904144 ERROR HASS: ------------------------------------------------------------
2023-04-09 11:22:42.910464 INFO nspanel-1: Sending MQTT Message: time~11:22~None
2023-04-09 11:22:42.914028 INFO HASS: Previous message repeated 1 times
2023-04-09 11:22:42.914344 ERROR HASS: Unexpected error during call_plugin_service()
2023-04-09 11:22:42.914587 ERROR HASS: Service: default.template.render Arguments: {'template': "- {{ states('sun.sun') }}"}
2023-04-09 11:22:42.914844 ERROR HASS: ------------------------------------------------------------
2023-04-09 11:22:42.915367 ERROR HASS: Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/appdaemon/plugins/hass/hassplugin.py", line 556, in call_plugin_service
    r = await self.session.post(api_url, json=data)
  File "/usr/local/lib/python3.10/site-packages/aiohttp/client.py", line 508, in _request
    req = self._request_class(
  File "/usr/local/lib/python3.10/site-packages/aiohttp/client_reqrep.py", line 305, in __init__
    self.update_host(url)
  File "/usr/local/lib/python3.10/site-packages/aiohttp/client_reqrep.py", line 364, in update_host
    raise InvalidURL(url)
aiohttp.client_exceptions.InvalidURL: /api/template

2023-04-09 11:22:42.915641 ERROR HASS: ------------------------------------------------------------
2023-04-09 11:22:42.921210 INFO nspanel-1: Sending MQTT Message: date~Sunday, 09. April 2023None
2023-04-09 11:22:42.924801 INFO nspanel-1: Sending MQTT Message: dimmode~10~100~6371
2023-04-09 11:22:42.928412 INFO nspanel-1: Sending MQTT Message: pageType~screensaver2
2023-04-09 11:22:42.931984 INFO nspanel-1: Sending MQTT Message: timeout~10
2023-04-09 11:22:42.982959 INFO nspanel-1: Sending MQTT Message: weatherUpdate~~~���~38066~Lunel~16.1��C~~~���~17299~Tasmota ANALOG Temperature1~27.0  ~~~���~17299~edf_pinst~472.73 W~~~���~17299~edf_price_tax_month~12.5040512 EUR/kWh~~~���~17299~Nuage~30 %~~~���~17299~Pluie~0 %~~~���~17299~Rafale~0 km/h~~~���~17299~Vent~7 km/h~~~���~17299~UV~6 ~~~���~2016~Niveau~Vert 
2023-04-09 11:22:42.990402 INFO nspanel-1: Sending MQTT Message: statusUpdate~���~65222~���~65222~~
2023-04-09 11:22:43.006367 INFO nspanel-1: MQTT callback for: {'topic': 'tele/nspanel01_E7F10C/RESULT', 'wildcard': None, 'payload': '{"nlui_driver_version":"8"}'}
2023-04-09 11:22:43.008212 INFO nspanel-1: Update Pre-Check sucessful Tasmota Driver Version: 8 Panel Version: 50
```

**Comment by lluisd** [comment]
I also have the same problem today. In my case the Appdaemon was auto updated last night before the errors. My panel was working yestarday at 3:00am and my auto update is from 4:00am

![image](https://user-images.githubusercontent.com/7629843/230765631-9b9f7607-192b-4f91-8d38-418c849e178a.png)
![image](https://user-images.githubusercontent.com/7629843/230765642-0c690d8e-d430-41ae-984e-ad3a5a85e488.png)

**Comment by joBr99** [comment]
k, so this also happens with perfectly valid templates and not only with None

```
2023-04-09 11:22:42.914587 ERROR HASS: Service: default.template.render Arguments: {'template': "- {{ states('sun.sun') }}"}
```

Sounds like an issue with the newest appdaemon version. Please try to downgrade your appdaemon version.

I'm using the version bundled with the homeassistant (4.2.3), newer versions are untested.



https://github.com/hassio-addons/addon-appdaemon/blob/main/appdaemon/requirements.txt

**Comment by ntls2000** [comment]
thanks lluisd , i go back to 4.2.3, it's ok for me ; problem with daemon 4.4.0

**Comment by soyxan** [comment]
Same problem here, not only with the dates on the screensaver, but also with the values of all entities.
Downgrading to 4.2.3 solves the issue.

**Comment by joBr99** [comment]
I cannot see anything related to templates in the release notes, would be nice if someone can create a minimal example to reproduce this bug and open an issue in the the appdaemon project. (https://github.com/AppDaemon/appdaemon/)

**Comment by joBr99** [comment]
k, there is acually already an issue for this

https://github.com/AppDaemon/appdaemon/issues/1747

**Comment by xiconfjs** [comment]
> 



> great, now this version is also out as HA Addon, going to make an example and open an issue upstream
> 
> @ HA OS Users do not upgrade AppDaemon from 0.11.0 to 0.12.0

is there a way for home assistant addon users which had auto-update enabled to downgrade?

**Comment by joBr99** [comment]
not sure about this, try to restore the backup

**Comment by irqnet** [comment]
> > 
> 
> > great, now this version is also out as HA Addon, going to make an example and open an issue upstream
> > @ HA OS Users do not upgrade AppDaemon from 0.11.0 to 0.12.0
> 
> is there a way for home assistant addon users which had auto-update enabled to downgrade?

Yeah, restore works. Just select the appdaemon out of your last full backup.

**Comment by TekFan** [comment]
It's not just the "none" on the screensaver for me.
I cannot get to the subpages when I touch the screen, the touch is registered however because the luminosity changes as usual, but then nothing else happens.
Which renders the whole thing almost useless.
However, I can still call the subpages via ESPHome or MQTT and then they work, but once the screensaver comes back, I'm stuck there.
Going back to previous version of appdaemon fixes the issue.
I don't know if I'm the only one seeing this because I don't see any report of this navigation problem.
As a side-note I'm on ESPhome, not Tasmota which is probably less common.

**Comment by joBr99** [comment]
it depends on your configuration, everything related to templates is broken. Using something like defaultCard with the buggy appdaemon version would cause thoose issues.

**Comment by TekFan** [comment]
That's it, I'm obviously using defaultCard.
Thanks for the info.
Let's be patient then.

**Comment by joBr99** [comment]
Easy fix: Restore your backup of the appdameon addon (0.11.0) until the issue is fixed in the addon, you've most likely have one

If you need to use the 0.12.0 version of the addon you can follow this to fix it:

1. SSH & Web Terminal from Home Assistant Community Add-ons
2. Configure the password or authorized_keys option
3. Disable Protection Mode in Addon Settings
4. Start the Addon
5. Open Web UI of the addon
6. Fix bug in appdaemon:
➜  ~ docker exec -it addon_a0d7b954_appdaemon /bin/bash
```
root@a0d7b954-appdaemon:/$ sed -i 's;"/api/template";f"{self.ha_url}/api/template";g' /usr/lib/python3.10/site-packages/appdaemon/plugins/hass/hassplugin.py
root@a0d7b954-appdaemon:/$ exit
```
7. Maybe enable protection mode again.

**Comment by esdcmc** [comment]
Not sure if it is related, but I also noticed that the following card grid entity was showing correct number (a temperature) with 0.11.0 and it is showing 'NONE' with 0.12.0. 

      - type: cardGrid
        entities:
          - entity: input_number.final_t_star_downstairs
            name: T goal
            icon: 'ha:{{ states("input_number.final_t_star_downstairs")}}'

I just proved this by restoring a previously saved 0.11.0 backup

**Comment by joBr99** [comment]
solved with Appdaemon 4.4.2 / 0.12.1
